tests/e2e/api-core-tests/bin/wc-api-tests.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+#
+# Run package scripts
+#
+
+# Script help
+usage() {
+	echo 'usage: npx wc-api-tests <script>'
+	echo 'scripts:'
+	echo '         test:api - run API tests'
```

Comment:
If this was changed to `test <group>` it would simplify the script and allow creating new groups without changing the script.

Reply:
That's a great idea--I prefer that! Do you think we should keep `test:api` in the main `package.json` file or keep that `test:api` then you pass in the group, something like this:

```shell
npx wc-api-tests test <group>
```

Versus keeping the script name and passing in the group, something like:

```shell
npx wc-api-tests test:api <group>
```

Reply:
That's a great idea! I've made the changes and removed the now extraneous `test:hello`.